pkg/nuctl/test/get_test.go

```diff
@@ -46,8 +46,11 @@ func (suite *GetTestSuite) SetupSuite() {
 }
 
 func (suite *GetTestSuite) TestGet() {
+
+	// Todo: add tests to check multiply get and delete's
```

Comment:
- This test seems like something that should be in the PR, not as a TODO - after all, this is the exact functionality this PR is adding. I think this test should remain unchanged, and a second `TestMultipleGet` should also exist.
- Terminology - the term to use is "multiple" (not "multiply")